packages/design-system/src/styles/components/_Alert.scss

```diff
@@ -46,6 +46,15 @@ $alert-link-color-focus: $color-primary-darkest;
     }
   }
 
+  /* stylelint-disable */
+  @media (-ms-high-contrast: active), (forced-colors: active) {
+    border-top: 1px solid windowText;
+    border-right: 1px solid windowText;
+    border-bottom: 1px solid windowText;
+    border-left: $alert-bar-size solid windowText;
```

Comment:
I wonder if this could be consolidated into 2 lines?
```css
border: 1px solid windowText;
border-left-style: $alert-bar-size;
```

Reply:
Ooh, I like that.